packages/date/src/test/index.js

```diff
@@ -642,6 +642,35 @@ describe( 'Moment.js Localization', () => {
 					'2023-04-30T13:00:00.000Z'
 				)
 			).toBe( '2 days ago' );
+
+			expect(
+				humanTimeDiff(
+					'2023-04-28T11:00:00.000Z',
+					'2023-04-28T13:00:00.000Z'
+				)
+			).toBe( '2 hours ago' );
+
+			expect(
+				humanTimeDiff(
+					'2023-04-26T13:00:00.000Z',
+					'2023-04-28T13:00:00.000Z'
+				)
+			).toBe( '2 days ago' );
```

Comment:
Are these two effectively the same as the above two `expect` statements?

Reply:
Whoops, good pick up. That was a leftover from hacking.